server/routes/applications.js

```diff
@@ -87,6 +87,33 @@ router.get('/manage-applications/:appId/view/:env', (req, res, next) => {
   });
 });
 
+router.get('/manage-applications/:appId/view/:env/:keyName', (req, res, next) => {
+  logger.info(`GET request to view a single application: ${req.path}`);
```

Comment:
Do any of the unite tests need updating?

Reply:
Added a unit test